### ISSUE TITLE
Insufficient contrast of .feature-text, .text-muted

### ISSUE THREAD

**Issue by fvsch** [yarnpkg/website#168]
Some medium-gray-on-white text doesn't pass WCAG contrast ratio tests:

```css
/* Home page mostly */
.feature-text {
    font-weight: 300;
    font-size: 1.1rem;
    color: #818a91;
}

/* e.g. on https://yarnpkg.com/en/docs/ */
.text-muted {
  color: #818a91 !important;
}
```

See for example this calculator:
http://webaim.org/resources/contrastchecker/

Additionally, I would advise against using `font-weight: 300` for anything but headings, since the thin text stroke increases the contrast issue even more. Font weights below normal/400 are harder to read for body copy in most fonts (with rare exceptions), and since in this project attempts to use the system fonts there is no way to control the fact that the `font-weight:300` results in readable body copy for all users on all platforms.

**Comment by thejameskyle** [comment]
I'm surprised the default bootstrap colors don't pass this... I just assumed it would

**Comment by fvsch** [comment]
Apparently they don’t.
I opened an issue there: https://github.com/twbs/bootstrap/issues/20952

**Comment by thejameskyle** [comment]
Improved in a99d54756a4de76eadf147d2e63ef043bad7351e
